frontend/src/app/module-blueprint/components/note-edit-panel/element-note-editor/element-note-editor.component.css

```
@@ -27,51 +42,77 @@
+  height: 12px;
+  border-radius: 2px;
+  overflow: hidden;
+}
+
+.temperature-scale svg {
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.control-slider ::ng-deep .p-slider {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)